### DISCUSSION TITLE
What should I do when this happens when I use PPOCRLabel?

### DISCUSSION THREAD

**Discussion by lingskr** [PaddlePaddle/PaddleOCR#13649] [Documents]
What should I do when this happens when I use PPOCRLabel?
![image](https://github.com/user-attachments/assets/402680ec-81a1-4eb2-ba83-886b3fd2fbfb)

**Comment by UserWangZz** [comment]
看着像系统cuda环境和paddle不一致导致的，可以检查一下paddle是否安装成功

**Reply by lingskr** [reply]
ok
